corehq/apps/cleanup/management/commands/rebuild_domain_cases.py

```diff
@@ -16,6 +17,9 @@ def handle(self, *args, **options):
         else:
             raise CommandError('Usage: %s\n%s' % (self.args, self.help))
 
+        if should_use_sql_backend(domain):
+            raise CommandError('This command only works for couch-based domains.')
```

Comment:
@dimagi/scale-team I wasn't sure if I should just be deleting these commands outright or doing something like this. I could see how they could arguably still be useful for couch domains but also happy to get rid of them